### ISSUE TITLE
Node.js crash with a TypeError

### ISSUE THREAD

**Issue by poying** [automerge/automerge#104]
```
> require('automerge')
TypeError: Cannot read property 'bind' of undefined
    at Object.byteToHex (/private/tmp/node_modules/automerge/dist/automerge.js:7689:79)
    at __webpack_require__ (/private/tmp/node_modules/automerge/dist/automerge.js:30:30)
    at Object.<anonymous> (/private/tmp/node_modules/automerge/dist/automerge.js:7648:11)
    at __webpack_require__ (/private/tmp/node_modules/automerge/dist/automerge.js:30:30)
    at Object.<anonymous> (/private/tmp/node_modules/automerge/dist/automerge.js:6341:12)
    at __webpack_require__ (/private/tmp/node_modules/automerge/dist/automerge.js:30:30)
    at Object.<anonymous> (/private/tmp/node_modules/automerge/dist/automerge.js:7227:12)
    at __webpack_require__ (/private/tmp/node_modules/automerge/dist/automerge.js:30:30)
    at /private/tmp/node_modules/automerge/dist/automerge.js:73:18
    at /private/tmp/node_modules/automerge/dist/automerge.js:76:10
```

__Node.js version__: `8.4.0`
__Automerge version__: `0.7.10`

**Comment by ept** [comment]
Hi @poying, thanks for your report. Is it possible for you to update Node.js? We test Automerge with versions [8.11.3](https://travis-ci.org/automerge/automerge/jobs/395454599), [9.11.2](https://travis-ci.org/automerge/automerge/jobs/395454600), and [10.5.0](https://travis-ci.org/automerge/automerge/jobs/395454601), and they all seem to be working fine. I'm not sure what changed between Node 8.4.0 and 8.11.3 that would cause this crash.

**Comment by poying** [comment]
@ept All the tests passed successfully on my environment, too, which is created by docker. I was also trying to run the same code with version 8.11.3, 9.11.2 and 10.5.0, but all met the same problem.

**Comment by ept** [comment]
Ok, I reproduced this issue, and found a fix. I have released Automerge version 0.7.11, which should fix the issue. Could you test it please @poying?

The issue does not depend on the version of node, but rather it depends on the version of the `uuid` package in use. For now I've fixed it by downgrading to version 3.1.0 of that package (see commit message on aca2c324e09048d1c2bb9b91247c0b2e5e01d7d8). However, there are some quite puzzling things about this issue.

The weirdest thing is that the issue does not occur when running the unit tests, even when the environment variable `TEST_DIST=1` is set (which causes us to load the webpack-packaged file rather than the source files from the `src` directory). The issue only occurs if you load `dist/automerge.js` directly in Node. I don't know why this would be the case; perhaps when Mocha is loaded, it messes around with some global objects such that the bug is not triggered?

I think when uuid version 3.3.0 is released, we should be able to upgrade to that version, and it should also work.

More generally, a better fix would perhaps be to have two different packaged versions of Automerge: one for Node, and one for browsers. The Node version should then use the file `lib/rng.js` in the uuid package (which uses the Node API `crypto.randomBytes`), rather than `lib/rng-browser.js`.

**Comment by poying** [comment]
@ept It works now, thanks!
